state/runtime/runtime.go

```diff
@@ -64,6 +65,7 @@ type Host interface {
 	GetStorage(addr types.Address, key types.Hash) types.Hash
 	SetStorage(addr types.Address, key types.Hash, value types.Hash, config *chain.ForksInTime) StorageStatus
 	SetState(addr types.Address, key types.Hash, value types.Hash)
+	SetNoBaseFee(noBaseFee bool)
```

Comment:
Consider to rename accordingly, in case field name gets changed.

Reply:
Done